src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 40 * days;
+
+  if (days >= 7) {
+    total = total - 50;
+  } else if (days >= 3) {
+    total = total - 20;
```

Comment:
move 40, 20 and 50 in the variables

Reply:
Seems that you didn't fix the previous comment( Please, check it one more time